src/cross_toml.rs

```diff
@@ -89,22 +89,22 @@ impl CrossToml {
 
     /// Returns the list of environment variables to pass through for `build`,
     pub fn env_passthrough_build(&self) -> &[String] {
-        &self.build.env.passthrough
+        self.build.env.passthrough.as_deref().unwrap_or(&[])
     }
 
     /// Returns the list of environment variables to pass through for `target`,
     pub fn env_passthrough_target(&self, target: &Target) -> &[String] {
-        self.get_vec(target, |e| &e.passthrough)
+        self.get_vec(target, |e| e.passthrough.as_deref().unwrap_or(&[]))
     }
 
     /// Returns the list of environment variables to pass through for `build`,
     pub fn env_volumes_build(&self) -> &[String] {
-        &self.build.env.volumes
+        self.build.env.volumes.as_deref().unwrap_or(&[])
     }
 
     /// Returns the list of environment variables to pass through for `target`,
     pub fn env_volumes_target(&self, target: &Target) -> &[String] {
-        self.get_vec(target, |e| &e.volumes)
+        self.get_vec(target, |e| e.volumes.as_deref().unwrap_or(&[]))
     }
```

Comment:
I think these should return opt as well, instead the decision to throw away this fact should be done at the caller

Reply:
I had those originally as Options, it makes the calling code a bit more difficult (specifically, the accumulation code). I can revert it back, however.

Reply:
I've made some changes in #678 that touches on that as well, fyi

Reply:
> I've made some changes in #678 that touches on that as well, fyi

Currently reviewing that as we speak, I'll look into that shortly.